VoodooGPIO/VoodooGPIOIntel.cpp

```diff
@@ -731,9 +747,11 @@ bool VoodooGPIOIntel::start(IOService *provider) {
 void VoodooGPIOIntel::stop(IOService *provider) {
     IOLog("%s::VoodooGPIO stop!\n", getName());
 
-    if (command_gate) {
-        workLoop->removeEventSource(command_gate);
-        OSSafeReleaseNULL(command_gate);
+    if (this->command_gate) {
+        if (this->workLoop) {
+            this->workLoop->removeEventSource(this->command_gate);
+        }
+        this->command_gate->release();
```

Comment:
`this->command_gate->release();` can be replaced with `OSSafeReleaseNULL(command_gate)`